Core/include/Acts/Surfaces/TrapezoidBounds.hpp

```diff
@@ -140,6 +144,7 @@ class TrapezoidBounds : public PlanarBounds {
  private:
   std::array<double, eSize> m_values;
   RectangleBounds m_boundingBox;
+  RotationMatrix2 m_rotMat{RotationMatrix2::Identity()};
```

Comment:
I guess `Eigen::Rotation2D<double>` would be more space efficient while `RotationMatrix2` may be more CPU efficient